src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -585,6 +589,18 @@ private bool ShouldSkip(ReplicationBatchItem item, OutgoingReplicationStatsScope
             if (ValidatorSaysToSkip(_pathsToSend) || ValidatorSaysToSkip(_destinationAcceptablePaths))
                 return true;
 
+            if (_shouldSkipSendingTombstones)
+            {
+                switch (item.Type)
```

Comment:
Can we unify this check with `_preventIncomingSinkDeletions` from the Hub?

Reply:
Done